### PR TITLE
Add Alexis Janvier profil

### PR DESCRIPTION
**Pull request by alexisjanvier**

### CODE REVIEW

**Comment by dadoonet** [comment]
@alexisjanvier Sorry for the delay. Could you update your PR (rebase on master or merge with master) as the js file changed a bit and is now in conflict. Note that the PR should only shows your addition and not other changes to existing talks. Thanks!

**Comment by alexisjanvier** [comment]
@dadoonet Done :smiley:

File: baggers.js [hunk 1]
```diff
@@ -14661,7 +14730,7 @@ var data = {
     },
     Caen: {
       name: 'Caen',
-      ville_img: 'img/villes/BBL_Caen.png',
+      ville_img: 'img/villes/BBL_Caen.jpg',
```
Comment (dadoonet) [inline baggers.js:14733]: I think this should be: `images/villes/caen.jpg` instead.

**Review by dadoonet — CHANGES_REQUESTED** [review]
Could you fix the link?

File: baggers.js (hunk 1 above)
Comment (alexisjanvier) [reply baggers.js:14733]: Ok, it's done
(But all other links to a city's picture are with the form `img/cities/BBL_cityName.imageType`)

Comment (dadoonet) [reply baggers.js:14733]: Hmmm. My bad... Please re-revert what I asked for. Although I think we don't use that image anymore.

Instead we should change http://www.brownbaglunch.fr/css/baggers.css file and add:

```css
.bagger-header.caen { background: url(../images/villes/caen.jpg) no-repeat center center; }
```

**Comment by alexisjanvier** [comment]
@dadoonet review applied

**Comment by dadoonet** [comment]
Yeah. Thanks. I'm sorry but we need to revert as you said. See https://github.com/brownbaglunch/bblfr_data/pull/291#discussion_r191188438

**Review by dadoonet — APPROVED** [review]
Great. Thanks!

**Pull request merged**